src/main/java/org/triplea/ui/TripleA.java

```diff
@@ -0,0 +1,104 @@
+package org.triplea.ui;
```

Comment:
I wonder if there should be a `client` package in there so that it is clear this `ui` is related to the client and not the lobby for example. `org.triplea.lobby` I suspect would be a package, odd for that to be side by side with the ui. We may also want to think more about how we organize ui classes. Instead of putting everything in a (horizontal layered) 'ui' bucket, instead we may find better cohesion if we group by feature. This way for example: `org.triplea.client.settings` would contain everything related to settings, including the UI.  The alternative we see has some scattering since some stuff is in settings, but the related UI is in a different package mixed together with other UIs.